notebooks/notebooks/iclr2022/win-rate-vs-visits.ipynb

```diff
@@ -0,0 +1,460 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.        for name in df2_points.index:
</pre><p>Do we expect exactly two names in the index? If so, should we assert that (i.e. we're not overwriting existing values for splusplus/perfect_modeling points)? If not, is the index guaranteed to be sorted appropriately?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>